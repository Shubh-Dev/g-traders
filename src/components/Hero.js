// import React from 'react';
// import './hero.css';
// // import ImageSlider from './ImageSlider';

// const slides = [
//   { url: 'http://localhost:3000/toilet-1.jpg' },
//   { url: 'http://localhost:3000/toilet-2.jpg' },
//   { url: 'http://localhost:3000/toilet-4.jpg' },
//   { url: 'http://localhost:3000/toilet-5.jpg' },
// ];

// const Hero = () => {
//   const containerStyles = {
//     width: '95%',
//     height: '280px',
//     margin: '0 auto',
//   };

//   return (
//     <div className="hero-container">
//       <div style={containerStyles}>
//         <ImageSlider slides={slides} />
//       </div>
//       <br />
//       <br />
//       <p className="hero-description">
//         Lorem ipsum dolor sit amet consectetur adipisicing elit.
//         Ipsam dolorem voluptas voluptate nisi accusamus doloribus
//         quibusdam vel error perferendis laborum corporis .
//       </p>
//       <div className="thick-line" />
//     </div>
//   );
// };

// export default Hero;
