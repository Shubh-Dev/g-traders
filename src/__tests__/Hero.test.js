import React from 'react';
import { render, screen } from '@testing-library/react';
import Hero from '../components/Hero';

describe('Hero', () => {
  it('renders correctly', () => {
    const { container } = render(<Hero />);
    expect(container).toMatchSnapshot();
  });

  it('renders the hero image', () => {
    render(<Hero />);
    const heroImage = screen.getByRole('img');
    expect(heroImage).toBeInTheDocument();
  });

  it('Loads the image on the page correctly', () => {
    render(<Hero />);
    const heroImage = screen.getByRole('img');
    expect(heroImage).toHaveAttribute(
      'src',
      'https://unsplash.com/photos/g51F6-WYzyU'
    );
  });
});
