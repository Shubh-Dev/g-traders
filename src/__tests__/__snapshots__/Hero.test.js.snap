// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero renders correctly 1`] = `
<div>
  <div
    class="hero-container"
  >
    <img
      alt="bathroom"
      src="https://unsplash.com/photos/g51F6-WYzyU"
    />
  </div>
</div>
`;
